processor/src/operations/io_ops.rs

```diff
@@ -181,10 +181,10 @@ where
         Ok(())
     }
 
-    /// Moves 8 elements from the head of the advice tape to memory via the stack.
+    /// Moves 8 elements from the head of the advice stack to memory via the stack.
```

Comment:
Let's update this to: "Moves 8 elements from the advice stack to memory via the operand stack."